python/doc/theory/data_analysis/quantile_estimation_wilks.rst

```diff
@@ -1,197 +1,90 @@
+:math:`X_{(k)}` is the :math:`k` -th maximum of :math:`(X_1, \dots, X_\sampleSize)` for :math:`1 \leq k \leq \sampleSize`. For
+example, :math:`X_{(1)} = \min (X_1, \dots, X_\sampleSize)` is the minimum
+and :math:`X_{(\sampleSize)} = \max (X_1, \dots, X_\sampleSize)` is the maximum. We have:
 
 .. math::
 
-   \begin{aligned}
-       \Prob{ Y^i \leq q_{Y^i}(\alpha)} = \alpha
-     \end{aligned}
+    X_{(1)} \leq X_{(2)} \leq \dots \leq X_{(\sampleSize)}
 
-If we have a sample
-:math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of :math:`N`
-independent samples of the random vector :math:`\vect{X}`,
-:math:`q_{Y^i}(\alpha)` can be estimated as follows:
 
--  the sample :math:`\left\{ \vect{x}_1,\ldots,\vect{x}_N \right\}` of
-   vector :math:`\vect{X}` is first transformed to a sample
-   :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` of the variable
-   :math:`Y^i`, using :math:`\underline{y} = h(\vect{x}_i,\vect{d})`,
+Estimation of the order statistics that will provide an upper bound to :math:`x_{\alpha}`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  the sample :math:`\left\{ y^i_1,\ldots,y^i_N \right\}` is then placed
-   in ascending order, which gives the sample
-   :math:`\left\{ y^{(1)},\ldots,y^{(N)} \right\}`,
+We assume that we have a sample of size :math:`\sampleSize` of :math:`X`, denoted by :math:`(x_1, \dots, x_\sampleSize)`.
+
+Given :math:`\alpha`, :math:`\beta`, and :math:`\sampleSize`, we seek for the smallest :math:`k` such that:
+
+.. math::
+    :label: EqOrderStat
 
--  this empirical estimation of the quantile is then calculated by the
-   formula:
+    \Prob{X_{(k)} \geq x_{\alpha}} \geq \beta
 
-   .. math::
+The order statistics :math:`X_{(k)}` follows the distribution which probability density function and cumulated distribution function
+are defined by:
 
-      \begin{aligned}
-            \widehat{q}_{y^i}(\alpha) = y^{([N\alpha]+1)}
-          \end{aligned}
+.. math::
+    :label: DistOrderStat
 
-where :math:`[N\alpha]` denotes the integral part of
-:math:`N\alpha`.
+    F_{X_{(k)}}(x) & = \sum_{i=k}^{\sampleSize} \binom{\sampleSize}{i}\left[F(x)\right]^i \left[1-F(x)
+    \right]^{\sampleSize-i} \\
+    p_{X_{(k)}}(x) & = (\sampleSize-k+1)\binom{\sampleSize}{k-1}\left[F(x)\right]^{k-1} \left[1-F(x)
+    \right]^{\sampleSize-k} p(x)
 
-For example, if :math:`N=100` and :math:`\alpha = 0.95`,
-:math:`\widehat{q}_Z(0.95)` is equal to :math:`y^{(96)}`, which is the
-:math:`5^\textrm{th}` largest value of the sample
-:math:`\left\{ y^i_1,\ldots,y^i_N \right\}`. We note that this
-estimation has no meaning unless :math:`1/N \leq \alpha \leq 1-1/N`. For
-example, if :math:`N=100`, one can only consider values of a to be
-between 1% and 99%.
+We notice that :math:`F_{X_{(k)}}(x) = \overline{F}_{(\sampleSize,F(x))}(k-1)` where :math:`F_{(\sampleSize,F(x))}` is the cumulated
+distribution function of the Binomial distribution :math:`\cB(\sampleSize,F(x))`.
```

Comment:
The notation $\overline{F}$ should be introduced too.
```suggestion
We notice that :math:`F_{X_{(k)}}(x) = \overline{F}_{(\sampleSize,F(x))}(k - 1)` where :math:`F_{(\sampleSize,F(x))}` is the cumulated
distribution function of the Binomial distribution :math:`\cB(\sampleSize,F(x))`
and :math:`\overline{F}_{(\sampleSize,F(x))}(k) = 1 - F_{(\sampleSize,F(x))}(k)` is the complementary cumulated distribution fonction (also named survival function in dimension 1).
```

Reply:
yes